README.md

```diff
@@ -56,7 +56,11 @@ fn main() {
     }
 }
 ```
-*(note: requires feature `derive`)*
+Add this to `Cargo.toml`:
+```toml
+[dependencies]
+clap = { version = "3.0.0-rc.7", features = ["derive"] }
```

Comment:
Yes I know, I put the explicit version number here, this will probably need to be fixed to `version = "3"` when it's released.

Reply:
Could you update `Cargo.toml` `[package.metadata.release]` to include a "replacement" for updating this?  Much better than forgetting when bumping to v3 or v4 :)

Reply:
I don't understand what you mean here. Do you have a pointer to some doc I could read to get it ?

Reply:
This is documented in the [cargo-release docs](https://github.com/crate-ci/cargo-release/blob/master/docs/reference.md#prepost-release-replacements) though the existing replacements also serve as an example.

Alternatively, I could pull your changes and make the change.

Reply:
Ah I see; I modified the PR accordingly but it's untested.

Reply:
I'll test it and update it accordingly